lib.php

```diff
@@ -97,6 +97,7 @@ function theme_boost_campus_get_pre_scss($theme) {
         'imageareaitemsmaxheight' => ['imageareaitemsmaxheight'],
         'showsettingsincourse' => ['showsettingsincourse'],
         'incoursesettingsswitchtoroleposition' => ['incoursesettingsswitchtoroleposition'],
+        'settingsaddablockposition' => ['settingsaddablockposition'],
```

Comment:
I think you can omit this line, as this is only needed for processing the setting in the SASS file. As far as I see, you do not touch the SASS file with this pull request.

(Note for eventually future pull requests: Please add this at the end of the array, as all new settings will be added at the end and there is no sorting according to the settings themselves.)